common/src/main/java/org/apache/celeborn/common/network/util/NettyUtils.java

```diff
@@ -106,4 +110,20 @@ public static PooledByteBufAllocator createPooledByteBufAllocator(
         allowCache ? PooledByteBufAllocator.defaultNormalCacheSize() : 0,
         allowCache ? PooledByteBufAllocator.defaultUseCacheForAllThreads() : false);
   }
+
+  public static PooledByteBufAllocator getShardPooledByteBufAllocator(AbstractSource source) {
+    synchronized (PooledByteBufAllocator.class) {
+      if (_allocator == null) {
+        CelebornConf conf = new CelebornConf();
```

Comment:
any chance to pass a conf instead create a new one?

Reply:
+1